extension/src/XLIFFDocument.ts

```diff
@@ -140,6 +140,30 @@ export class Xliff implements XliffDocumentInterface {
         fs.writeFileSync(path, bom + this.toString(replaceSelfClosingTags, formatXml), encoding);
     }
 
+    /**@description Returns map of source string and translated targets.
+     * @summary description
+     * @returnType {Map<string, string[]>}
+     */
+    public translationMap(): Map<string, string[]> {
+        let transMap = new Map<string, string[]>();
+        this.transunit.filter(t => !isNullOrUndefined(t.target) && t.targetsHasTextContent()).forEach(unit => {
+            if (!isNullOrUndefined(unit.target)) {
+                if (!transMap.has(unit.source)) {
+                    transMap.set(unit.source, [unit.target[0].textContent]); //FIXME: Use index 0???
```

Comment:
remove FIXMEs? target[0] is fine, isn't it?

Reply:
Yes, this should be the default behavior, for the moment at least.